src/software/simulated_tests/avoid_collisions.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+from software.py_constants import *
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_types,
+    create_validation_geometry,
+)
+
+
+class RobotDoesNotCollide(Validation):
```

Comment:
Nit: This validation name may be slightly confusing. Could be more descriptive if the name is plural (i.e. robotics not colliding with each other, rather than colliding with some general obstacle)
```suggestion
class RobotsDoNotCollide(Validation):
```